### ISSUE TITLE
FE | Profile 508-defect-4: The legend element should be the first child of the fieldset for notification preferences

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#31449]
#### VFS team name
Authenticated Experience

#### VFS product name
Notification Preferences

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
The `legend` element for all notification settings should be the first child of the `fieldset`, otherwise it will not announce or group properly.

### Link, screenshot or steps to recreate
1. Log in to staging using test user 202
2. Navigate to notification settings
3. Inspect the legend and confirm it is not the first child of the fieldset

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
The current behavior works with `aria-label`, so this is a defect-4 ticket that the team can take on if they're interested in making their HTML more lightweight and less `aria` dependent (first rule of `aria` is not to use `aria`) To do so:

1. Move the `legend` element so it is the first child of the `fieldset`. This makes sure it is announced as a group when a user tabs to a radio button.
2. Delete the `aria-labels`. Since the `legend` will now be announced, the `aria-labels` that artificially append the `legend` to the radio buttons will no longer be needed.
3. Test with Angela to confirm it still parses correctly (should read out as Notify me by text, 1 of 2, Board of Veterans' Appeals hearing reminder)

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by shiragoodman** [comment]
@joshkimux can you please update the title to this ticket? Thanks!

**Comment by jcatergovy** [comment]
@SKasimow We found where they wanted the change but it messes with the styling and there's no easy way to fix this.  Adam and I feel that this ticket is best left alone.

**Comment by SKasimow** [comment]
@joshkimux - fyi. Any thoughts or should we close this?

**Comment by joshkimux** [comment]
@SKasimow I believe this should be fixed within the DST component soon, let's keep this ticket open until that's officially merged

**Comment by SKasimow** [comment]
@joshkimux - sounds good; any idea on timing?

**Comment by joshkimux** [comment]
[It looks like it has been merged! ](https://github.com/department-of-veterans-affairs/component-library/pull/535) @SKasimow

**Comment by SKasimow** [comment]
Thanks @joshkimux!
